core/lib/workarea/configuration/administrable_options.rb

```diff
@@ -1,18 +1,14 @@
 module Workarea
   module Configuration
     class AdministrableOptions < ActiveSupport::InheritableOptions
-      def method_missing(name, *args)
+      def [](name)
         static_config = super
         return static_config if static_config.present? || static_config.to_s == 'false'
         return static_config unless check_fieldsets?(name)
 
         Configuration::Admin.instance.send(name)
       end
 
-      def respond_to_missing?(name, include_private)
```

Comment:
this is also the exact same defintiion in `AS::InheritableOptions` so no need to redefine it here.